src/Tasks/ResolveComReferenceCache.cs

```diff
@@ -19,8 +19,6 @@ namespace Microsoft.Build.Tasks
     /// 
     /// This is an on-disk serialization format, don't change field names or types or use readonly.
```

Comment:
Is this comment stale now too? Don't we handle serialization manually agnostic of names?

Reply:
Yes, we do. Fixed by removing these comments as well, thank you!